.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -0,0 +1,43 @@
+name: 🐛 Bug report
+description: Create a report to help us improve
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue.
+
+        This issue tracker is for bugs and issues found within bun.
+        If you require more general support please reach out to our [discord server](https://bun.sh/discord).
+
+        Please fill in as much of the following form as you're able.
+  - type: input
+    attributes:
+      label: Version
+      description: Output of `bun -v`
+  - type: input
+    attributes:
+      label: Platform
+      description: |
+        UNIX: output of `uname -a`
```

Comment:
Instead of UNIX as a general platform I think it would be more user friendly to put `Linux/MacOS` instead

Reply:
I am not really sure about that. Because that command can be run on another UNIX-based OS family. So I think using that would be the best. Plus also people who are using this should be understood that Linux/MacOS is part of UNIX family.